profiles/signals.py

```diff
@@ -0,0 +1,48 @@
+from anymail.exceptions import AnymailError
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django_ilmoitin.utils import send_notification
+from sentry_sdk import capture_exception
+
+from .consts import REPRESENTATIVE_CONFIRMATION_DEGREE
+from .models import LegalRelationship
+
+
+@receiver(post_save, sender=LegalRelationship)
+def relationship_saved_handler(sender, instance, created, **kwargs):
+    confirmed = instance.confirmation_degree != REPRESENTATIVE_CONFIRMATION_DEGREE[0][0]
+
+    # Only notify person whose confirmation is needed
+    if created and not confirmed:
+        try:
+            send_notification(
+                instance.representative.email,
+                "relationship_confirmation_needed",
+                instance.get_notification_context(),
+                instance.representative.language,
+            )
+        except (OSError, AnymailError) as e:
+            print(e)
```

Comment:
🤔 weird that flake8 check did not catch this

Reply:
Yeah, damn flake8 leaving in these pesky print statements 😆